packages/theming/default-theme/src/defaultColors.ts

```diff
@@ -31,7 +32,7 @@ export function paletteFromFabricColors(p: FabricWebPalette, isInverted?: boolea
     defaultStateBackground: p.neutralLighterAlt,
```

Comment:
Is there a possibility to code share not platform specific code between defaultColors and implementations - most of the code is same across the 2 files..

Reply:
I agree there is a lot of duplication. I'm not sure if there is a good way to do this though, from what I've seen other files that got split up by platform usually have a lot of duplication

Reply:
Generally it would be handled by generating part of the object in a function defined in a forked file while the common parts stay in a non-forked file.

Reply:
Think I'm going to leave this file for now, maybe in the future we can try to consolidate our component token files to share more code. For this file specifically it sounds like we want to merge some theming stuff anyway